test/e2e/apps/cronjob.go

```diff
@@ -261,6 +261,26 @@ var _ = SIGDescribe("CronJob", func() {
 		framework.ExpectNoError(err, "Failed to delete CronJob %s in namespace %s", cronJob.Name, f.Namespace.Name)
 	})
 
+	ginkgo.It("should set the cronjob-scheduled-timestamp annotation on a job", func(ctx context.Context) {
+		ginkgo.By("Creating a cronjob")
+		cronJob := newTestCronJob("concurrent", "*/1 * * * ?", batchv1.AllowConcurrent,
+			nil, nil, nil)
+		cronJob, err := createCronJob(ctx, f.ClientSet, f.Namespace.Name, cronJob)
+		framework.ExpectNoError(err, "Failed to create CronJob in namespace %s", f.Namespace.Name)
+
+		ginkgo.By("Ensuring CronJobsScheduledAnnotation annotation exists on the newely created job")
+		err = waitForJobsAtLeast(ctx, f.ClientSet, f.Namespace.Name, 1)
+		framework.ExpectNoError(err, "Failed to ensure a job exists in namespace %s", f.Namespace.Name)
+
+		jobs, err := f.ClientSet.BatchV1().Jobs(f.Namespace.Name).List(ctx, metav1.ListOptions{})
+		gomega.Expect(jobs.Items[0].Annotations).Should(gomega.HaveKey(batchv1.CronJobScheduledTimestampAnnotation),
```

Comment:
I know we have a unit test for this, but we could at least test that it is possible to parse the timestamp